client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -373,30 +390,50 @@ export default React.createClass( {
 
 	unverifiedDomainsCanManage() {
 		const domains = this.getDomains().filter( domain => domain.isPendingIcannVerification && domain.currentUserCanManage );
+		const severity = domains.some( ( { registrationMoment } ) =>
+			registrationMoment && moment( registrationMoment )
+				.add( 2, 'days' )
+				.isAfter( moment() ) ) ? 'is-info' : 'is-error';
```

Comment:
If you don't pass any parameters to `isAfter` it will default to the current time ([docs](http://momentjs.com/docs/#/query/is-after/)). I like explicit parameters in most cases, but in this one, I think omitting `moment()` leads to a line that flows better.